.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    "env": {
+        "es6": true,
+        "browser": true
+    },
+    "extends": "eslint-config-digitalbazaar",
+};
```

Comment:
Is there any reason why we're using 4 spaces here instead of 2? Is that the preferred eslintrc.js style?

Reply:
we need a linter for our linter configs!

Reply:
exactly